rust/automerge/src/storage/convert/op_as_docop.rs

```diff
@@ -135,7 +135,8 @@ impl<'a> Iterator for OpAsDocOpSuccIter<'a> {
     type Item = DocOpId;
 
     fn next(&mut self) -> Option<Self::Item> {
-        if let Some(s) = self.op.succ().get(self.offset) {
+        // FIXME
```

Comment:
What does this refer to?

Reply:
Ah - good call - ill expand upon the FIXME - nth() is no longer fast b/c  the iterator changed - fixing wasn't trivial b/c the iterator is anonymous